appraisal/ruby-3.3.rb

```diff
@@ -131,7 +133,6 @@
   gem 'roda', '>= 2.0.0'
   gem 'semantic_logger', '~> 4.0'
   gem 'sidekiq', '~> 7'
-  gem 'sneakers', '>= 2.12.0'
```

Comment:
Should this change be propagated to the other appraisal files as well? Noticed the others still list `sneakers` under contrib